src/networking.c

```diff
@@ -2874,7 +2879,10 @@ sds catClientInfoString(sds s, client *client) {
         " redir=%I", (client->flags & CLIENT_TRACKING) ? (long long) client->client_tracking_redirection : -1,
         " resp=%i", client->resp,
         " lib-name=%s", client->lib_name ? (char*)client->lib_name->ptr : "",
-        " lib-ver=%s", client->lib_ver ? (char*)client->lib_ver->ptr : ""));
+        " lib-ver=%s", client->lib_ver ? (char*)client->lib_ver->ptr : "",
+        " tot-net-in=%U", client->net_input_bytes,
+        " tot-net-out=%U", client->net_output_bytes,
+        " tot-cmds=%U", client->commands_processed));
```

Comment:
In the PR description, they are called `tot-input` and `tot-output`.

I like the shorter `tot-input` and `tot-output` better.

If by any chance the terms "input" and "output" are uncluear (the input of the client is the output of the server and vice versa), we could call them "tot cmd bytes" and "tot reply bytes", though those are longer. With proper documentation, the shorter ones are good.

Reply:
Yeah, I thought tot-input was confusing because it could overlap with input buffers or some other type of input. I don't really like `tot-cmd-bytes`, but I would be okay with `tot-bytes-in` and `tot-bytes-out`.

Reply:
`tot-bytes-in` and `tot-bytes-out` don't look less confusing than `tot-input` and `tot-output`.

`tot-cmd-bytes` and `tot-reply-bytes` would miss the replication flow, which is also counted.

If we want them to be clear, we may need to specify the flow direction, like `bytes-from-client` and `bytes-to-client`, but these seems too long?

Reply:
Ok, @valkey-io/core-team please give input on naming ^. I'm generally okay with either `tot-*` or `tot-bytes-*`.

Reply:
Yeah, in and out are clear to me.

So "net" is added to clarify it isn't something like total memory of input buffer, or something like that?

Then I'm fine with tot-net-in, tot-net-out. I'm also fine without tot-input, tot-output. Or even net-in, net-out. (Most of those client fields can't be understood without docs anyway.)

The comment below for tot-net-in is wrong though: :grin:

    tot-net-in   // Total bytes the client has read
    tot-net-out  // Total bytes the server has sent to the client

Reply:
> The comment below for tot-net-in is wrong though: 😁
I got myself confused ...

Reply:
I vote for `tot-net-in`, it is clear to me and can also be considered as an abbr for "total_net_input_bytes", aligning with `INFO` reply.

Reply:
Very good point @soloestoy.

Reply:
I have another thought:

In fucntion  writeToClient(), c->net_output_bytes += totwritten; and in function readQueryFromClient(), c->net_input_bytes += nread; The words are fine becuase it happens in server side.

But in client side function catClientInfoString, it should reflect client view，i suggest to update as following:

" tot-send=%U", client->net_input_bytes,
" tot-receive=%U", client->net_output_bytes,
" tot-cmds=%U", client->commands_processed));

Thus, client->net_input_bytes (server received bytes) match with client send (tot-send),  and client->net_output_bytes and tot-receive match as well.

Reply:
we have some bad experiences about that, we used "read_bytes" instead of "input_bytes". But it is ambiguous, since from the server's view that means how many datas it **reads** from network via client, but from the user's view "read_bytes" means the data they **read** from server.

the word "input" and "output" is clearer IMO.

Reply:
Yes, @soloestoy I agree. Even CLIENT INFO is from the point of view for the server. It's just the I/O layer of the server. Anyway, the documentation is important here.

Reply:
I have no objection to keep current name. Thanks